force-app/main/default/classes/VersionCleanupBatchJobController_TEST.cls

```diff
@@ -129,18 +156,38 @@ private with sharing class VersionCleanupBatchJobController_TEST {
         }
     }
 
+    /**************************************************************************************************************************************************
+     * @description Stub class to simulate the response from VersionCleanupBatchJobService to return a true for runCourseConnectionBackfillJob method .
+     **************************************************************************************************************************************************/
+    private class STUB_runCourseConnectionBackfillJobServiceValid extends VersionCleanupBatchJobService {
+        public override Boolean runCourseConnectionBackfillJob() {
+            return true;
+        }
+    }
+
     /**************************************************************************************************************************************************
      * @description Stub class to simulate the response from VersionCleanupBatchJobService to return a true for runCourseDescriptionCopyBatchJob method .
      **************************************************************************************************************************************************/
+
     private class STUB_CopyCourseDescriptionJobServiceValid extends VersionCleanupBatchJobService {
         public override Boolean runCourseDescriptionCopyBatchJob() {
             return true;
         }
     }
 
+    /***************************************************************************************************************************************************
+     * @description Stub class to simulate the response from VersionCleanupBatchJobService to throw an error for runCourseConnectionBackfillJob method
+     ***************************************************************************************************************************************************/
+    private class STUB_runCourseConnectionBackfillJobServiceInvalid extends VersionCleanupBatchJobService {
+        public override Boolean runCourseConnectionBackfillJob() {
+            throw new AuraHandledException(Label.BatchJobRunningProblem);
+        }
+    }
+
     /***************************************************************************************************************************************************
      * @description Stub class to simulate the response from VersionCleanupBatchJobService to throw an error for runCourseDescriptionCopyBatchJob method
      ***************************************************************************************************************************************************/
+
```

Comment:
extra spacing here. Surprising prettier is not catching this.

Reply:
I think this is just git hub, the file looks good @
![Screen Shot 2021-05-20 at 11 27 06 AM](https://user-images.githubusercontent.com/49822508/119030235-78390800-b95e-11eb-95ab-4330a4010171.png)